src/__tests__/fixtures/color-case.compressed.css

```diff
@@ -1 +1 @@
-p{color:#AA93FE;background-color:#AFE}
\ No newline at end of file
+p{color:#AA93FE;background:#FFF url(/ImagesDir/loader.gif) center center no-repeat}
```

Comment:
This is also related to the issue #43 where a hex color + data URI is all transformed.